### ISSUE TITLE
[Accessibility] Cannot use challenge coding editor with screen reader: When focused, screen reader doesn't detect existing text in the editor.

### ISSUE THREAD

**Issue by choiyoung7** [freeCodeCamp/freeCodeCamp#14682]
Challenge [Headline with the h2 Element](https://www.freecodecamp.com/challenges/headline-with-the-h2-element#?solution=%3Ch1%3EHello%20World%3C%2Fh1%3E%0A%3Ch2%3ECatPhotoApp%3C%2Fh2%3E) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64; Trident/7.0; .NET4.0C; .NET4.0E; .NET CLR 2.0.50727; .NET CLR 3.0.30729; .NET CLR 3.5.30729; rv:11.0) like Gecko</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```html
<h1>Hello World</h1>
<h2>CatPhotoApp</h2>
```

MSIE: 11.873.10586.0
Firefox: 53.0
Chrome: 58.0.3029.81
Jaws: 18.0.2738
NVDA: 2017.1

Steps:
1. Start screen reader, (Jaws/NVDA).
2. Open a challenge using MSIE/Firefox/Chrome.
3. Put focus to the code editor.
4. Try to edit the text using screen reader.
Actual: Screen reader doesn't detect any character as system character moves.
Expected: The user should be able to edit the code using screen reader. Screen reader should behave the same as any other multi-line edit field, E.G. notepad or gmail message.

**Comment by raisedadead** [comment]
Hi I am closing this in favor of a pre-existing issue https://github.com/freeCodeCamp/freeCodeCamp/issues/12431

Thanks for your report.
